Classes/Domain/Model/DocumentFormField.php

```diff
@@ -55,6 +55,12 @@ class DocumentFormField extends AbstractFormElement
      */
     protected $maxInputLength;
 
+    /**
+     * Embargo field option
+     *
+     * @var bool
+     */
+    protected $embargo = false;
```

Comment:
Why is embargo an option on document form fields?

Reply:
Embargo is not needed as option on document form fields anymore. It was needed with the automatic embargo. So this is a leftover, because the automatic embargo option was rendered as an additional checkbox.

Reply:
If it is not needed anymore, it should be removed.